client/reader/discover/helper.jsx

```diff
@@ -8,14 +8,12 @@ var find = require( 'lodash/find' ),
 /**
  * Internal Dependencies
  */
-var config = require( 'config' ),
-	userUtils = require( 'lib/user/utils' ),
+var userUtils = require( 'lib/user/utils' ),
 	readerRoute = require( 'reader/route' );
 
 module.exports = {
 	isEnabled: function() {
-		return config.isEnabled( 'reader/discover' ) &&
-			userUtils.getLocaleSlug() === 'en';
+		return userUtils.getLocaleSlug() === 'en';
```

Comment:
separate PR but what happens if this is `en-gb` or `en-au` or `en-ca`... not sure we actually support those at the moment.